src/test/java/com/google/javascript/clutz/partial/legacy_namespace_forward_declare.js

```diff
@@ -0,0 +1,13 @@
+goog.module('legacy.namespace.forward.declare');
+goog.module.declareLegacyNamespace();
+
+const MissingForwardDeclare = goog.forwardDeclare('missing.forward.Declare');
```

Comment:
Does it matter if 'missing.forward.Declare' is a goog.module or goog.provide? Is this fix assuming one or the other?

Reply:
Our usual methodology for determining that will still work.  In this case, since we passed in no depgraphs, it defaults to goog.module.